### PR TITLE
Makes sures to pass application feature flags to Glimmer compiler

### PR DESCRIPTION
**Pull request by twokul**

### CODE REVIEW

File: lib/broccoli/glimmer-app.ts [hunk 1]
```diff
@@ -342,6 +347,12 @@ Please run the following to resolve this warning:
     return appTree;
   }
 
+  public getGlimmerEnvironment(): any {
+    let config = this.project.config(this.project.env);
+
+    return config.GlimmerENV || config.EmberENV;
```
Comment (rwjblue) [inline lib/broccoli/glimmer-app.ts]: Should we fallback, or deep merge?

File: lib/broccoli/glimmer-app.ts [hunk 2]
```diff
@@ -364,9 +375,10 @@ Please run the following to resolve this warning:
     return maybeDebug(compiledTypeScriptTree, 'typescript-output');
   }
 
-  private compiledHandlebarsTree(srcTree) {
+  private compiledHandlebarsTree(srcTree, options) {
     let compiledHandlebarsTree = new GlimmerTemplatePrecompiler(srcTree, {
-      rootName: this.project.pkg.name
+      rootName: this.project.pkg.name,
+      GlimmerENV: options.GlimmerENV
```
Comment (rwjblue) [inline lib/broccoli/glimmer-app.ts]: Why pass `options` and not `GlimmerENV` directly?

File: lib/broccoli/glimmer-app.ts (hunk 1 above)
Comment (twokul) [inline lib/broccoli/glimmer-app.ts]: my thinking was that it's unlikely someone would specify two variables in `config/environment.js` so I decided to fallback.

File: lib/broccoli/glimmer-app.ts (hunk 2 above)
Comment (twokul) [inline lib/broccoli/glimmer-app.ts]: I could change

**Pull request merged**